fixtures-processor/src/main/kotlin/com/theblueground/fixtures/ParameterValueGenerator.kt

```diff
@@ -233,21 +236,27 @@ internal class ParameterValueGenerator {
     }
 
     private fun generateSealedValue(
-        randomize: Boolean,
+        kspArguments: KspArguments,
         parameter: ProcessedFixtureParameter.SealedParameter,
     ): String {
-        val sealedEntry = if (randomize) {
-            parameter.entries.random()
+        val entries = parameter.entries.filter { it.isObject || it.isFixture }
+        require(entries.isNotEmpty()) {
+            "At least one sealed data class annotated with @Fixture is required in: ${parameter.type.copy(nullable = false)}"
```

Comment:
This message might not fully account for scenarios where we have only objects, which is valid. Should we update it to reflect this case for better accuracy? 🤔